test/test.js

```diff
@@ -19,4 +19,16 @@ describe('is-relative-path', function () {
     assert.ok(!isRelative('foo'));
     assert.ok(!isRelative('foo/bar/car/baz.js'));
   });
+
+  it('throws on empty input', function () {
+    assert.throws(function () {
+      isRelative();
+    }, function (err) { return (err instanceof TypeError); });
```

Comment:
I'm open to it. I still think testing error messages is important to:

1. Make sure our usage of neat-contract is correct
2. Assert an expected behavior of the lib. 

Knowing that we're using a tested, 3rd party lib for errors under the hood is an implementation detail. Plus, if neat-contract changes its generated error message, that breaks the messaging we agreed on for is-relative-path